src/functions/sendCandidateResults/framework/request-scheduler.ts

```diff
@@ -33,13 +33,20 @@ export class RequestScheduler implements IRequestScheduler {
     @inject(TYPES.IStatusUpdater) private statusUpdater: IStatusUpdater,
   ) {
     this.limiter = new bottleneck({
-      maxConcurrent: null,                 // No limit on concurrent requests
-      minTime: 0,                          // No time waited between each request
-      highWater: 250,                      // Maximum of 250 requests in the queue (max batch size)
-      strategy: bottleneck.strategy.BLOCK, // Ignore any additions to the queue when we reach the max batch size
-      reservoir: 25,                       // Amount of jobs the queue can perform at the start of the queue
-      reservoirRefreshInterval: 1000,      // How often to add new jobs to the queue (every second)
-      reservoirRefreshAmount: 25,          // How many jobs to add to the queue each refresh
+      // No limit on concurrent requests
+      maxConcurrent: null,
+      // No time waited between each request
+      minTime: 0,
+      // Maximum of 250 requests in the queue (max batch size)
+      highWater: this.getNumberFromEnv('NOTIFY_BATCH_SIZE') || 250,
+      // Ignore any additions to the queue when we reach the max batch size
+      strategy: bottleneck.strategy.BLOCK,
+      // Amount of jobs the queue can perform at the start of the queue
+      reservoir: this.getNumberFromEnv('NOTIFY_REQUESTS_PER_BATCH') || 25,
+      // How often to add new jobs to the queue (every second)
+      reservoirRefreshInterval: 1000,
+      // How many jobs to add to the queue each refresh
+      reservoirRefreshAmount: this.getNumberFromEnv('NOTIFY_REQUESTS_PER_BATCH') || 25,
```

Comment:
Should the reservoir and reservoirRefreshAmount always be the same?  
Is it possible we would want these to be different values... e.g the reservoir is 25 but we only want to refresh 15?
If so then they should be two separate environment variables.

Reply:
Perhaps someone with more knowledge on these fields can weigh in but looking at the tars upload lambda, they are set from the same env var there too - 
https://github.com/dvsa/mes-result-processing-service/blob/develop/src/functions/tarsUpload/framework/adapter/upload/RateLimitDecoratingTARSUploader.ts

Reply:
From memory, the reservoir is the number of concurrent requests you want to run on initialization and the reservoirRefreshAmount is the number of requests you want to run every interval following the initialization.

In our case, we only want to run 25/second and there is no difference for us between the first and second runs. From the docs =

> reservoir is a counter decremented every time a job is launched, we set its initial value to 100. Then, every reservoirRefreshInterval (60000 ms), reservoir is automatically updated to be equal to the reservoirRefreshAmount (100).

There are some examples of this under Reservoir Intervals in the docs - https://www.npmjs.com/package/bottleneck

Reply:
The docs says:
`reservoirRefreshAmount` is the value to set `reservoir` to when `reservoirRefreshInterval` is in use.